packages/main/package.json

```diff
@@ -82,8 +82,7 @@
   },
   "dependencies": {
     "@ui5/webcomponents-base": "0.7.0",
-    "@ui5/webcomponents-core": "0.7.0",
-    "lit-html": "^1.0.0"
+    "@ui5/webcomponents-core": "0.7.0"
```

Comment:
is there a change at all? `,` ?

Reply:
that's why we enforce trailing commas in js, to avoid such diffs. This is json however, so trailing comma not allowed.